Kudu.Core/Environment.cs

```diff
@@ -151,7 +151,15 @@ public string RepositoryPath
         {
             get
             {
-                return FileSystemHelpers.EnsureDirectory(_repositoryPath);
```

Comment:
I think it may make sense to add a FileSystemHelpers.EnsureDirectoryIgnoreAccessExceptions() helper, and then just call that from all 4 places.

Reply:
That makes sense. Thanks, I will update this PR.